src/index.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/dist/output.css" rel="stylesheet" />
+  </head>
+  <body class="home">
```

Comment:
home block is not a good name, it confuse, we don 't know what it is doing and where it belong, i suggest you find anaother name, like content-wrapper